platform/util/platform-util-unavailableurls/pom.xml

```diff
@@ -24,6 +26,11 @@
     <packaging>jar</packaging>
     <dependencies>
         <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```

Comment:
❓ What is this dependency being used for? Is it production or test code? If the latter, it should be marked as `scope` test.

❓ In either case, have we run OWASP over this module yet to ensure it has no vulnerabilities?